README.md

```diff
@@ -7,44 +7,14 @@
 
 ## PEL parser
 
-The pel.peltool.peltool module parses PEL files and prints the resulting PEL
-fields in JSON.  It will load and run the various component parsers if they are
-available.
+- Peltool parser is a Python module that takes binary PEL data as input and displays the output in JSON format.
+- Peltool parser can be used on both BMC and non-BMC environments.
 
 ### Usage
 
 The tool can parse a single PEL or a directory of PELs at once.
 
-The following command writes the output to standard out.
-
-```bash
-$ python3 -m pel.peltool.peltool -f <PEL file>
-```
-
-The following commands parse every PEL in the directory passed in and write the
-output to `<output directory>/<original filename>.<PEL EID>.json`.  It will
-skip files that aren't PELs.
-
-```bash
-$ python3 -m pel.peltool.peltool -d <directory> -o <output directory>
-```
-
-```bash
-$ python3 -m pel.peltool.peltool -d <directory> -o <output directory> -e '.extracted'
-```
-
-If the `-o` option isn't specified, the files will be written to the same
-directory the PELs are in.
-
-The `-x` option can be used to tell peltool to delete the original file after
-it is parsed.
-
-The `-e` option can be used to only look at files with a specific extension.
-
-The `-p` option can be used to skip loading any plugin modules.
-
-The `-s` option can be used to only parse PELs if they are for serviceable
-errors, while the `-n` option will only parse PELs for non-serviceable ones.
+For more peltool usuage info please refer modules/pel/peltool/README.md.
```

Comment:
Markdown supports hyper links so please use that to point the peltool README file.

For example, 

```
[here](./modules/pel/peltool/README.md).
```